src/language/syntax.md

```diff
@@ -0,0 +1,55 @@
+# Syntax
+
+```
+P ::= [i]\*                     Program items
+i ::= array!(t, μ, n)           Array type declaration where n is a natural number
+    | nat_mod!(t, n)            Modular integer
+    | fn f([d]+) -> μ b         Function declaration
+    | type t = enum { [c(μ)]+ } Enum declaration (with constructors)
+    | type t = struct([μ]+)     Struct declaration (without named fields)
+    | const x: μ = e            Constant declaration
+    | use crate::*              External crate import
+d ::= x: τ                      Function argument
+μ ::= unit|bool|u8|U8|i8|I8...  Base types
```

Comment:
We have to define the `U8...` (secret integers) somewhere.

Reply:
They are handled internally in the language as regular functions, but I can add them to the syntax.